lib/stsci/tools/asnutil.py

```diff
@@ -14,7 +14,9 @@
 from astropy.io import fits
 import numpy as N
 import os.path, time
+from distutils.version import LooseVersion
 
+ASTROPY_VER_GE13 = LooseVersion(astropy.__version__) >= LooseVersion('1.3')
```

Comment:
The most fool-proof way is to *not* depend on version checking but it looks ugly:

```python
try:
    with catch_and_hide_the_warning:  # pseudocode
        fits_ptr.write('blahblah', overwrite=True)
except:
    fits_ptr.write('blahblah', clobber=True)
```